### PR TITLE
Foreign key checkbox for empty/drop in table operation

### PR DESCRIPTION
**Pull request by nisargjhaveri**

### CODE REVIEW

**Comment by lem9** [comment]
Yes, a function would do.

**Comment by nisargjhaveri** [comment]
Updated, please check now.

File: js/functions.js [hunk 1]
```diff
@@ -1692,6 +1692,14 @@ function getForeignKeyCheckbox() {
     return html;
 }
 
+function getJSConfirmCommonParam(elem) {
```
Comment (nisargjhaveri) [inline js/functions.js]: A better name?

Comment (lem9) [inline js/functions.js]: This is a fine function name.

**Pull request merged**